README.md

```diff
@@ -3,15 +3,16 @@
 [![GitHub release](https://img.shields.io/github/release/iMicknl/ha-tahoma.svg)](https://GitHub.com/iMicknl/ha-tahoma/releases/)
 [![Open in Visual Studio Code](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/iMicknl/ha-tahoma/)
 
-# Somfy TaHoma - Home Assistant
+# Overkiz (by Somfy) - Home Assistant
```

Comment:
Maybe we could remove the parentheses?

Reply:
What would be the benefit? In the core integration, we call everything Overkiz and thus it makes sense to rebrand here as well. For the core integration, auto discovery will help us enable every Overkiz hub user :-).

Reply:
@vlebourl You mean `Overkiz` or `Overkiz by Somfy`?

Reply:
I just thought `Overkiz by Somfy` is better than `Overkiz (by Somfy)`. But I won't fight for it, I'll accept the majority's opinion.

Reply:
I can settle with both, so @tetienne you can make the final decision. 😉. I just wanted to not have Somfy to prominent visible, since it can scare users that their hubs won't be supported.

Reply:
That’s why I wrote it between braces to make it less "mandatory". We can also ask to the community on the forum. What do yo think?